jobs/cloud_controller_ng/spec

```diff
@@ -613,10 +613,15 @@ properties:
   cc.opi.enabled:
     description: "Enable the amazing eirini component"
     default: false
-
+  cc.opi.opi_staging:
+    description: "Enable the amazing eirini staging"
+    default: false
   cc.opi.url:
     description: "Eirini Endpoint"
     default: ""
+  cc.opi.cc_uploader_url:
+    description: "URL of cc uploader"
+    default: http://cc-uploader.service.cf.internal:9090
```

Comment:
Can you consume these from this link?
https://github.com/cloudfoundry/capi-release/blob/e9f4ef1b903dda22e2eea03343900a734c2e4017/jobs/cc_uploader/spec#L26-L29

Reply:
Yes, that should work.
We could keep this property, like diego's: https://github.com/cloudfoundry/capi-release/blob/1bc9db2c5cd5cca004ff674b4b7844c38130caa5/jobs/cloud_controller_ng/spec#L1001
I could then open a PR to consume the link in the templates

Reply:
@gdankov @viovanov 

Diego's property exists because it predated the BOSH link that the cc-uploader provides and to retain backwards compatibility it was kept. Unless I'm missing something, I don't think there is a reason why `cc.opi.cc_uploader_url` can't just reuse the same logic thats already in the template:

https://github.com/cloudfoundry/capi-release/blob/e9f4ef1b903dda22e2eea03343900a734c2e4017/jobs/cloud_controller_ng/templates/cloud_controller_ng.yml.erb#L413

We might proceed with the PR as-is and remove this property (and just consume the link) before the next capi-release unless there is a compelling reason to maintain both.